dir.props

```diff
@@ -373,11 +373,7 @@
   <PropertyGroup Condition="'$(OutputRid)' != ''">
     <PackageTargetRid>$(OutputRid)</PackageTargetRid>
     <PackageTargetRid Condition="'$(OutputRid)' == 'osx.10.11-x64'">osx.10.10-x64</PackageTargetRid>
-    <PackageTargetRid Condition="'$(OutputRid)' == 'rhel.7.0-x64'">rhel.7-x64</PackageTargetRid>
-    <PackageTargetRid Condition="'$(OutputRid)' == 'rhel.7.1-x64'">rhel.7-x64</PackageTargetRid>
-    <PackageTargetRid Condition="'$(OutputRid)' == 'rhel.7.2-x64'">rhel.7-x64</PackageTargetRid>
-    <PackageTargetRid Condition="'$(OutputRid)' == 'rhel.7.3-x64'">rhel.7-x64</PackageTargetRid>
-    <PackageTargetRid Condition="'$(OutputRid)' == 'rhel.7.4-x64'">rhel.7-x64</PackageTargetRid>
+    <PackageTargetRid Condition="'$(OutputRid.StartsWith(&quot;rhel.7.&quot;))' and '$(OutputRid.EndsWith(&quot;-x64&quot;))'">rhel.7-x64</PackageTargetRid>
```

Comment:
You can change this to `$(OutputRid.StartsWith('rhel.7.')) and $(OutputRid.EndsWith('-x64'))`

Reply:
Hm. I am sure I tried that and it failed with errors. Let me try again.

Reply:
Just tried it:

    /home/omajid/core-setup/dir.props(376,23): error MSB4092: An unexpected token "rhel" was found at character position 26 in condition "'$(OutputRid.StartsWith('rhel.7.'))' and '$(OutputRid.EndsWith('-x64'))'". [/home/omajid/core-setup/src/pkg/deps/deps.csproj] [/home/omajid/core-setup/build.proj]

Reply:
Remove the single quotes around the outside of the property. `'$(OutputRid.StartsWith('rhel.7.'))'` -> `$(OutputRid.StartsWith('rhel.7.'))`

Reply:
Thanks!